app/helpers/application_helper.rb

```diff
@@ -2,7 +2,9 @@
 
 module ApplicationHelper
   def back_link_url(back = url_for(:back))
-    return check_answers_path if session[:form_complete] && request.path != check_answers_path
+    if session[:form_complete] && [check_answers_path, itt_provider_path].exclude?(request.path)
+      return check_answers_path
+    end
```

Comment:
I think this isn't working correctly; found a :bug: bug:

Review page -> Click "Change teacher training provider" -> `itt_provider_path` -> Click back -> :exclamation: `awarded_qts_path`

Instead if should go to `check_answers_path`.

Having a think how to fix this...

Reply:
🤔

Reply:
Added a fix for this.